### PR TITLE
feat: [IAC-376]: Update the IACM icon

### PR DESCRIPTION
**Pull request by willd0yle**
IACM icon has now been changed. This is so that similar modules have matching colours

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Review by stuart-mcdonnell — APPROVED** [review]

File: packages/icons/src/iacm-blue.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...317-17.73089c-.0056-.7569-.7906-1.2592-1.5667-1.00255l-12.43534 3.52497c-.51814.17135-.86892.63763-.85958 1.14263zm2.57541 4.06574c0-.2595.21036-.4699.46986-.4699h1.5502v-2.13873c0-.06584.04426-.12345.10788-.14041l3.74537-.99878v-1.3914c0-.2595.2104-.46987.4699-.46987h2.9367c.2595 0 .4699.21037.4699.46987v2.93669c0 .2595-.2104.46987-.4699.46987h-2.9367c-.2595 0-.4699-.21037-.4699-.46987v-1.41638l-3.71328.99022c-.00908.00242-.01541.01065-.01541.02006v2.13873h1.26189c.2595 0 .4699.2104.4699.4699v2.9367c0 .2595-.2104.4698-.4699.4698h-1.26189v2.08c0 .0094.00632.0177.01541.0201l3.71328.9902v-1.3577c0-.2595.2104-.4698.4699-.4698h2.9367c.2595 0 .4699.2103.4699.4698v2.9367c0 .2595-.2104.4699-.4699.4699h-2.9367c-.2595 0-.4699-.2104-.4699-.4699v-1.4501l-3.74537-.9988c-.06362-.0169-.10788-.0745-.10788-.1404v-2.08h-1.5502c-.2595 0-.46986-.2103-.46986-.4698zm.93973.4699v1.9969h1.99698v-1.9969zm5.87338-3.1423v-1.99695h1.9969v1.99695zm-.9398 2.6724c0-.2595.2104-.4699.4699-.4699h2.9367c.2595 0 .4699.2104.4699.4699v2.9367c0 .2595-.2104.4698-.4699.4698h-2.9367c-.2595 0-.4699-.2103-.4699-.4698zm.9398.4699v1.9969h1.9969v-1.9969zm0 7.1361v-1.997h1.9969v1.997z" fill="#0278d5" fill-rule="evenodd"/></svg>
```
Comment (knagurski) [inline packages/icons/src/iacm-blue.svg:1]: Can you remove the `width` and `height` attributes on these SVGs? Otherwise they won't scale

Comment (willd0yle) [reply packages/icons/src/iacm-blue.svg:1]: Will do

**Review by stuart-mcdonnell — APPROVED** [review]

File: packages/icons/src/iacm-blue.svg (hunk 1 above)
Comment (willd0yle) [reply packages/icons/src/iacm-blue.svg]: Done

**Review by chrisblakely01 — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
👍🏻

**Pull request merged**